netutils/argtable3/Make.defs

```diff
@@ -0,0 +1,27 @@
+############################################################################
+# apps/netutils/argtable3/Make.defs
```

Comment:
argtable3 is a generic library not specific to net, how about we put it under apps/system/?

Reply:
Agree, I will put it to apps/system/, thank you for your suggestion.